docs/publishing-repo.md

```diff
@@ -1,20 +1,58 @@
 When you're ready to tell the world about your GitHub repo, consider the following:
 
-* Check that the "About" section and the contents of `README.md` are both up to date. People will see these sections prominently when they click a link to your repository on GitHub.
-    * Include a short, one-line summary and link to your paper or outputs in the "About" section. Click the cog next to "About" and add these details.
-    * In `README.md`, include:
-        * A link to the preprint/journal article associated with the repo.
-        * A link to the published outputs in your project workspace on the jobs site.
+## About Section
+Check that the "About" section is up to date.
+People will see this section prominently when they view your repository on GitHub.
+
+Include a short, one-line summary and link to your paper or outputs in the "About" section.
+Click the cog next to "About" and add these details.
+
+
+## README
+As of the 24th October 2022 we have changed how we managed repo READMEs.
+You can ignore this section if you created your repo after that date.
+
+If you created your repo before this date then you will have the older README format, which expected you to update the file as your project progressed.
+We have now moved to keeping that information in your project on the Jobs site.
+
+The [current README template](https://github.com/opensafely/research-template/blob/main/README.md) has the updated version for new repos.
+You can update your repo by replacing the relevant sections in your own README (typically just the first section) with the content from the template.
+You will also need to replace the placeholder strings in the "View on OpenSAFELY" URL:
```

Comment:
question: Does this not correctly get updated? There is [some kind of script](https://github.com/opensafely/research-template/blob/7061202277ea9bc70837894db066673f8b03c637/.github/workflows/setup.yaml which is supposed to do this, I think.

Reply:
It does for _new_ repos, this paragraph is attempting to explain how an existing repo could update their README to the new version.  Any how I could better sign post that it's for repos made before the new template went live?

Reply:
Ah, right, now I got it; it's when the reader is copy-pasting!

> You can update your repo by replacing the relevant sections in your own README (typically just the first section) with the content from the template.

Maybe:

> The [current README template](https://github.com/opensafely/research-template/blob/main/README.md) has the updated version for new repos.
> To update your repository's README to match this new README:
> 1. Copy the content from the template repository's latest README (typically [just the first section](https://github.com/opensafely/research-template/blob/main/README.md?plain=1#L1-L3)).
> 2. Manually edit those changed sections into your research repository's README.
> 3. You will need to replace the two placeholder strings in the README's "View on OpenSAFELY" URL to point to your repository:
>
> * `${GITHUB_REPOSITORY_OWNER}` is the GitHub organisation your repo lives under, likely `opensafely`.
> * `${GITHUB_REPOSITORY_NAME}` is the name of your repo.

Something like that would have made it clearer for me. I'm not entirely sure that linking the lines directly is ideal, because they could change though. And there might be a more concise way of putting that.

Reply:
thoughts:

* That escaped URL is a little painful to edit manually. It's *almost*, but not quite, like an "update README/template" action would be useful.
* Likewise, it might almost be useful versioning the research template, so you can link directly to a version.

Reply:
> a little painful to edit manually

Understatement of the year 😂

We're not sure if we'll update readmes to save the researchers time yet, but I wanted the docs to help any researcher who happens to read them between now and when we make a decision.  There's every possibility that we generate a list of every readme in the org and some lucky soul gets to go fix them all!